widgets/matplotlib/simulation/element.py

```diff
@@ -93,6 +89,8 @@ def __init__(self, parent, element: Element, parent_tab,
             isotope_superscript = general.digits_to_superscript(
                 str(element.isotope))
             button_text = isotope_superscript + " " + element.symbol
+        elif element.sequence:
+            button_text = element.symbol + '#' + str(element.sequence)
```

Comment:
Use f-string here like you did in `get_prefix`, the same applies for the above line